Users/Tutorial.rst

```diff
@@ -170,11 +170,14 @@ Thats all nice and well but we also have a Makefile for our project we
 want to check. So let us introduce another feature of our configuration
 syntax: *sections*.
 
-The line ``[Default]`` specifies that everything below will belong to
-the Default section. If nothing is specified, a setting will implicitly
-belong to this section.
+The line ``[cli]`` implies that everything below belongs to the "cli"
+section. This section handles all the settings that are entered via the
```

Comment:
You could actually say "This *special* section" (because it really is special in coala's setting system)